web/moduly/finance.php

```diff
@@ -89,11 +81,8 @@
     <?php } ?>
     <ul class="seznam-bez-okraje">
     <li class="poznamka">Při pozdější platbě tě systém dne <strong>1.7</strong> (příp. 16.7. při pozdější přihlášce)<strong> automaticky odhlásí</strong>.</li>
```

Comment:
Tady by bylo dobré použít limit místo hardcode (prvního) datumu, když už v systému je. Dá se pak bavit i o druhém datu, ale to necháváme stranou na https://trello.com/c/ssQR3UQ0/413-proces-2-odhla%C5%A1ov%C3%A1n%C3%AD-finan%C4%8Dn%C4%9B-p%C5%99ihla%C5%A1ovac%C3%AD-fuckup-text.

Reply:
Kvůli opravě chyby s dynamikou jsem přidělal i druhé datum